data/archive/bapc/1994.yaml

```diff
@@ -1,10 +1,10 @@
 contest:
   name: NKP 1994
-  date: Amsterdam (UvA)
+  date: Amsterdam (UvA), October 8
   type: nkp
   year: 1994
 resources:
-  problems:
+  problems: nkp94.pdf
```

Comment:
I think problem G is missing from NKP 1994? Problems F and H are there, though

Reply:
There is is no problem G in the folder for NKP 94. This page might have been lost, or the jury forgot how to do their alphabets.